multistep/commonsteps/step_http_server.go

```diff
@@ -70,6 +126,8 @@ func (s *StepHTTPServer) Run(ctx context.Context, state multistep.StateBag) mult
 func (s *StepHTTPServer) Cleanup(multistep.StateBag) {
 	if s.l != nil {
 		// Close the listener so that the HTTP server stops
-		s.l.Close()
+		if err := s.l.Close(); err != nil {
+			log.Printf("Failed closing http server on port %d: %v", s.l.Port, err)
```

Comment:
Good question, this error should be temporary and won't prevent Packer from working. We check if ports are already open for new calls to `Listen`. Finally, when Packer terminates: the connection will be auto closed on most systems. Though on some system it could take some time and cause a FS leak if it happened A LOT of times. Yup this should be at least be shown in the UI.